recipes/spaln/patch-adddef_h

```diff
@@ -1,11 +0,0 @@
---- src/adddef.h.orig	2018-08-20 21:20:46.757620508 +0200
-+++ src/adddef.h	2018-08-20 21:20:53.425569375 +0200
-@@ -39,7 +39,7 @@
- #define ERROR	(-1)
```

Comment:
@pvanheus why have you removed this one?

Reply:
oops, that snuck in from other work - with the `spaln` update that patch file is no longer needed but I forgot too remove it in the associated PR, now it got into this one instead.

Reply:
This has been reverted with `e9b29580edbf21fd9d52be304682bb2d71c7a44c` (thanks to the guide on https://stackoverflow.com/questions/36168839/how-to-remove-commits-from-a-pull-request).